scout/adapter/mongo/case.py

```diff
@@ -565,9 +565,9 @@ def update_case(self, case_obj):
             {'_id': case_obj['_id']}
         )
 
```

Comment:
```suggestion

```
        rerun_requested = case.get('rerun_requested', False)
        if case.get('rerun_requested'):
            updated_status = old_case.get('status')
            if case.status in ['archived', 'active']:
                updated_status = 'inactive'
            rerun_requested = False
```

Reply:
and then ```rerun_requested``` in the ```$set ``` operation below obviously?

Reply:
Oh I see now that you've suggested the solution,thanks! Im going to look into it later today

Reply:
..and it doesn't have to be today - you were planning not to work all day, remember? 😄

Reply:
I need to do some matchmaker testing but it won't take too long, and this one is quite short as well!

Reply:
But if you have time ypi could always take over this PR!